sympy/utilities/pytest.py

```diff
@@ -199,15 +199,7 @@ def warns(warningcls, **kwargs):
 else:
     XFAIL = py.test.mark.xfail
     slow = py.test.mark.slow
-
-    def SKIP(reason):
-        def skipping(func):
-            @functools.wraps(func)
-            def inner(*args, **kwargs):
-                skip(reason)
-            return inner
-
-        return skipping
+    SKIP = py.test.mark.skip
```

Comment:
This is I think unrelated to the other changes but I needed it to use pytest while working on this. I was getting collection errors from this. It shouldn't affect the normal test runner as this is in the USE_PYTEST block.